src/components/CreateCanvasDialog.tsx

```diff
@@ -44,6 +44,9 @@ export default function CreateCanvasDialog({
 			open={isOpen}
 			onOpenChange={() => {
 				onClose();
+				if (Object.values(form.formState.errors).length === 0) {
```

Comment:
It would be better if you created variable or function that describes why you want to stop the form.reset() call